version.go

```diff
@@ -28,6 +28,7 @@ import "runtime/debug"
 
 const (
 	mainModule = "github.com/gocql/gocql"
```

Comment:
Note: this string must remain as-is until we rename packages.  That won't happen in 1.7.0, however, since the goal of 1.7.0 is just to get an ASF-branded release containing the current contents of the repo out (with some minor modifications).

Reply:
Please add a note that the string must remain as-is until v2 release as a code comment.

Reply:
Done.